src/constants/ipfs.ts

```diff
@@ -10,9 +10,6 @@ export const RESTRICTED_IPFS_GATEWAY_HOSTNAME =
 
 export const INFURA_IPFS_API_BASE_URL = 'https://ipfs.infura.io:5001'
 
-// Heavily rate-limited, don't use unless you have to.
-export const PUBLIC_PINATA_GATEWAY_HOSTNAME = 'gateway.pinata.cloud'
```

Comment:
don't see a need for this anymore, now that the infura ipfs gateway is public and open - does the same thing